tests/api.spec.ts

```diff
@@ -382,49 +383,4 @@ describe('Requests to public API', () => {
     },
     TEST_TIMEOUT,
   );
-
-  // TODO: this test is being skipped for now since the /report flow hasn't been fully rolled out and it can't succeed for now
-  it.skip('test successful report workflow', async () => {
```

Comment:
This test didn't seem to make much sense based on the others we already have, so I decided to remove it, but it can be discussed